deepspeech.cpp

```diff
@@ -7,14 +7,18 @@ extern "C" {
             ModelState* model;
 
         public:
-            ModelWrapper(const char* aModelPath)
+            ModelWrapper(const char* aModelPath, int *errorOut)
             {
-                DS_CreateModel(aModelPath, &model);
+                model = nullptr;
```

Comment:
On MacOSX I'm getting these errors on build:

```
deepspeech.cpp:12:25: error: use of undeclared identifier 'nullptr'
deepspeech.cpp:20:29: error: use of undeclared identifier 'nullptr'
deepspeech.cpp:166:21: error: use of undeclared identifier 'nullptr'
deepspeech.cpp:174:25: error: use of undeclared identifier 'nullptr'
deepspeech.cpp:197:21: error: use of undeclared identifier 'nullptr'
deepspeech.cpp:205:21: error: use of undeclared identifier 'nullptr'
deepspeech.cpp:212:21: error: use of undeclared identifier 'nullptr'
```

Here's my cpp version:

```
$ cpp --version
Apple clang version 11.0.0 (clang-1100.0.33.12)
Target: x86_64-apple-darwin18.7.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

Reply:
Ah, I forgot that `nullptr` is a C++11 feature, and it isn't enabled by default. I've set `CXXFLAGS` now. Mind trying again?